### PR TITLE
[zh]translate kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md into Chinese

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/34185

Do not have chinese docs: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1/

so need to translatekubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md into Chinese

### CODE REVIEW

**Comment by windsonsea** [comment]
Good job. Related to #34733.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 1]
```diff
@@ -0,0 +1,1387 @@
+---
+api_metadata:
+  apiVersion: "autoscaling/v1"
+  import: "k8s.io/api/autoscaling/v1"
+  kind: "HorizontalPodAutoscaler"
+content_type: "api_reference"
+description: "水平 pod 自动缩放器的配置。"
+title: "HorizontalPodAutoscaler"
+weight: 11
+auto_generated: false
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:10]: remove this line.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 2]
```diff
@@ -0,0 +1,1387 @@
...
+
+<!--
+- **apiVersion**: autoscaling/v1
+
+
+- **kind**: HorizontalPodAutoscaler
+-->
+- **apiVersion**: autoscaling/v1
+
+
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:51]: remove this useless empty line.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 3]
```diff
@@ -0,0 +1,1387 @@
...
+<!--
+- **spec** (<a href="{{< ref "../workload-resources/horizontal-pod-autoscaler-v1#HorizontalPodAutoscalerSpec" >}}">HorizontalPodAutoscalerSpec</a>)
+
+  behaviour of autoscaler. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status.
+-->
+- **spec** (<a href="{{< ref "../workload-resources/horizontal-pod-autoscaler-v1#HorizontalPodAutoscalerSpec" >}}">HorizontalPodAutoscalerSpec</a>)
+
+  自动缩放器的规约。
+  更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status.
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:72]: 这种“裸的”超链接前面要有空格才能被 markdown 解析。请彻查。


```suggestion
  更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status.
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 4]
```diff
@@ -0,0 +1,1387 @@
...
+<hr>
+
+<!--
+- **maxReplicas** (int32), required
+
+  upper limit for the number of pods that can be set by the autoscaler; cannot be smaller than MinReplicas.
+-->
+- **maxReplicas** (int32)，必填
+
+  自动扩缩器可以设置的 pod 数量上限；
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:103]: 译文中 pod 一律拼写为 Pod。请彻查。

```suggestion
  自动扩缩器可以设置的 Pod 数量上限；
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 5]
```diff
@@ -0,0 +1,1387 @@
...
+  不能小于 MinReplicas。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:104]: 这里的 minReplicas 是说另一个字段，不是数据结构，首字母要小写。类似问题请彻查。

```suggestion
  不能小于 minReplicas。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 6]
```diff
@@ -0,0 +1,1387 @@
...
+
+<!--
+- **scaleTargetRef** (CrossVersionObjectReference), required
+
+  reference to scaled resource; horizontal pod autoscaler will learn the current resource consumption and will set the desired number of pods by using its Scale subresource.
+-->
+- **scaleTargetRef** (CrossVersionObjectReference)，必填
+
+  参考缩放资源；
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:113]:
```suggestion
  对被扩缩资源的引用；
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 7]
```diff
@@ -0,0 +1,1387 @@
...
+  水平 pod 自动缩放器将了解当前的资源消耗，并使用其 Scale 子资源设置所需的 pod 数量。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:114]:
```suggestion
  水平 Pod 自动缩放器将了解当前的资源消耗，并使用其 scale 子资源设置所需的 Pod 数量。
```

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 8]
```diff
@@ -0,0 +1,1387 @@
...
+  **CrossVersionObjectReference 包含足够的信息来让你识别出所引用的资源。**
+
+  <!--
+  - **scaleTargetRef.kind** (string), required
+
+    Kind of the referent; More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"
+  -->
+  - **scaleTargetRef.kind** (string)，必填
+
+    所指对象的种类；
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:130]:
```suggestion
    被引用对象的类别；
```

下同。

Comment (twilight0620) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md]: I have modified them. Thanks for your review.

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 9]
```diff
@@ -0,0 +1,1381 @@
...
+  -->
+  <a name="CrossVersionObjectReference"></a>
+  **CrossVersionObjectReference 包含足够的信息来让你识别出所引用的资源。**
+
+  <!--
+  - **scaleTargetRef.kind** (string), required
+
+    Kind of the referent; More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"
+  -->
+  - **scaleTargetRef.kind** (string)，必填
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:125]: 此行之前要加入一个空行，保证排版正确。

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 10]
```diff
@@ -0,0 +1,1381 @@
...
+
+    被引用对象的类别；
+    更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"
+
+  <!--
+  - **scaleTargetRef.name** (string), required
+
+    Name of the referent; More info: http://kubernetes.io/docs/user-guide/identifiers#names  
+  -->
+  - **scaleTargetRef.name** (string)，必填
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:135]: 此行之前要加入一个空行，保证排版正确。

File: content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md [hunk 11]
```diff
@@ -0,0 +1,1381 @@
...
+
+    被引用对象的名称；
+    更多信息： http://kubernetes.io/docs/user-guide/identifiers#names
+
+  <!--
+    - **scaleTargetRef.apiVersion** (string)
+  
+      API version of the referent
+  -->
+  - **scaleTargetRef.apiVersion** (string)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:145]: 此行之前要加入一个空行，保证排版正确。

Comment (twilight0620) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:145]: 注释上面是有一行空行的，我在ide里面看格式是没问题的，需要在注释下面再加一行空行么？

Comment (tengqm) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:145]: 对。二层列表条目前面必须有空行，不然渲染出来的缩进是不对的。

Comment (twilight0620) [reply content/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:148]: 好的，已修改。

**Comment by tengqm** [comment]
@twilight0620 Can you help mark conversations resolved?

**Comment by tengqm** [comment]
/approve
Preview looks clean: https://deploy-preview-34747--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1/

**Comment by twilight0620** [comment]
> @twilight0620 Can you help mark conversations resolved?

you mean the comments? all the comments have been resolved.

**Comment by twilight0620** [comment]
> > > @twilight0620 Can you help mark conversations resolved?
> > 
> > 
> > you mean the comments? all the comments have been resolved.
> 
> I mean, if they are resolved, please **mark them as resolved**. Based on "outdated" status, I cannot tell whether the feedbacks are resolved.

ok, I have marked them.

**Pull request merged**